The data directory is hard coded so the config.xml is not found.

$OpenBSD$
--- src/lib/common.py.orig	Sat Nov 15 21:26:10 2014
+++ src/lib/common.py	Sat Nov 15 21:26:48 2014
@@ -8,7 +8,7 @@ try:
     from defs import *
 except:
     APPVERSION = "0.7.3"
-    datadir = "/usr/share"
+    datadir = "/usr/local/share"
 
 # Application info
 APPNAME = "Radio Tray"
